src/types/onyx/Policy.ts

```diff
@@ -68,18 +69,101 @@ type TaxRatesWithDefault = {
     taxes: TaxRates;
 };
 
-// These types are for the Integration connections for a policy (eg. Quickbooks, Xero, etc).
-// This data is not yet used in the codebase which is why it is given a very generic type, but the data is being put into Onyx for future use.
-// Once the data is being used, these types should be defined appropriately.
-type ConnectionLastSync = Record<string, unknown>;
-type ConnectionData = Record<string, unknown>;
-type ConnectionConfig = Record<string, unknown>;
+type ConnectionLastSync = {
+    successfulDate?: string;
+    errorDate?: string;
+    isSuccessful: boolean;
+    source: 'DIRECT' | 'EXPENSIFYWEB' | 'EXPENSIFYAPI' | 'AUTOSYNC' | 'AUTOAPPROVE';
+};
+
+type Account = {
+    glCode?: string;
+    name: string;
+    currency: string;
+    id: number;
+};
+
+type Employee = {
+    id: number;
+    firstName: string;
+    lastName: string;
+    name: string;
+    email: string;
+};
+
+type Vendor = {
+    id: number;
+    name: string;
+    currency: string;
+    email: string;
+};
+
+type ConnectionData = {
```

Comment:
Remind me, are we planning on using this for all accounting integrations? Others don't use the exact same field, so maybe we can call this `QuickBooksOnlineConnectionData` for now, and then in the future we can use it as `data: QuickBooksOnlineConnectionData | XeroConnectionData | [etc...]`. Same for `ConnectionConfig`.

Reply:
that's a good idea will do that